### PR TITLE
#2761 Add workaround for leading / in command line support documentation

### PR DESCRIPTION
**Pull request by etraisnel2**
Change-Id: Idc0ada73a041bce75c4de5d044398f12d110d0a6

### CODE REVIEW

File: doc/plugins/org.polarsys.capella.commandline.doc/html/19. Command Line Support/19.1. Core Mechanism and Applications.mediawiki [hunk 1]
```diff
@@ -65,6 +65,7 @@ The core mechanism provides the following parameters:
 [[Image:Images/19.1. Core Mechanism & Applications_html_3c78e175.png|48x48px]]
 |
 <ul>
+<li>When using custom scripts, some arguments may be modified in live. This can cause issues when using paths starting with a "/".If it occurs, try removing the leading "/" from the paths used as arguments.</li><br/>
```
Comment (pdulth) [inline doc/plugins/org.polarsys.capella.commandline.doc/html/19. Command Line Support/19.1. Core Mechanism and Applications.mediawiki:68]: Can you add a space before `.If it`

**Review by pdulth — CHANGES_REQUESTED** [review]

File: doc/plugins/org.polarsys.capella.commandline.doc/html/19. Command Line Support/19.1. Core Mechanism and Applications.mediawiki (hunk 1 above)
Comment (etraisnel2) [reply doc/plugins/org.polarsys.capella.commandline.doc/html/19. Command Line Support/19.1. Core Mechanism and Applications.mediawiki]: Done

File: doc/plugins/org.polarsys.capella.commandline.doc/html/19. Command Line Support/19.1. Core Mechanism and Applications.html [hunk 2]
```diff
@@ -84,6 +84,8 @@ <h2 id="Overview">Overview</h2>
 				</td>
 				<td>
 					<p><ul>
+<li>When using custom scripts, some arguments may be modified in live. This can cause issues when using paths starting with a "/" .If it occurs, try removing the leading "/" from the paths used as arguments.</li>
```
Comment (pdulth) [inline doc/plugins/org.polarsys.capella.commandline.doc/html/19. Command Line Support/19.1. Core Mechanism and Applications.html:87]: :) 
`starting with a "/" .If it occurs`
Shall be:
`starting with a "/". If it occurs`

**Review by pdulth — CHANGES_REQUESTED** [review]

File: doc/plugins/org.polarsys.capella.commandline.doc/html/19. Command Line Support/19.1. Core Mechanism and Applications.html (hunk 2 above)
Comment (etraisnel2) [reply doc/plugins/org.polarsys.capella.commandline.doc/html/19. Command Line Support/19.1. Core Mechanism and Applications.html]: Oops,
done correctly this time I hope :p

**Pull request merged**